src/tools/fuzzing.h

```diff
@@ -327,28 +327,31 @@ class TranslateToFuzzReader {
     }
     SmallVector<Type, 2> options;
     options.push_back(type); // includes itself
-    TODO_SINGLE_COMPOUND(type);
-    switch (type.getBasic()) {
-      case Type::anyref:
-        if (wasm.features.hasReferenceTypes()) {
-          options.push_back(Type::funcref);
-          options.push_back(Type::externref);
-          if (wasm.features.hasExceptionHandling()) {
-            options.push_back(Type::exnref);
+    // TODO: interesting uses of typed function types
+    // TODO: interesting subtypes of compound types
+    if (type.isBasic()) {
+      switch (type.getBasic()) {
+        case Type::anyref:
```

Comment:
Not related to this PR, but just a question: we don't have `dataref` yet right?

Reply:
I don't think so - I don't even know what that is. Which proposal is it from?

Reply:
It seems to be a reference for a heap type `data`, which is "the common supertype of all compound data types, like struct and array types and possibly host-defined types, for which casts are allowed". It is in the [MVP doc](https://github.com/WebAssembly/gc/blob/master/proposals/gc/MVP.md).

Reply:
I see, thanks. I missed that in the proposal. So far I haven't seen a use or need for it in code samples.